README.md

```diff
@@ -588,6 +588,12 @@ For a real-life example, you can check my personal dots:
 - `module` is auto-loaded. No need to specify
 - `keys` spec is [different](#%EF%B8%8F-lazy-key-mappings)
 
+With packer `wants`, `requires` and `after` can be used to manage dependencies.
+With most of the lua dependencies this isn't necessary. They can be installed just like normal plugins (even with `lazy=true`) and will be loaded when other plugins need them.
```

Comment:
sorry to nitpick here, but maybe start the sentence with `With lazy, this isn't needed for most of the lua dependencies.`